assets/css/styles.css

```diff
@@ -258,6 +262,8 @@ summary {
 }
 
 #lose-tie {
```

Comment:
change all the #lose-tie slecetors to class selectors instead of IDs. It not hiding when the end game.